src/shared/bg_parse.cpp

```diff
@@ -2328,10 +2320,6 @@ void BG_ParseMissileDisplayFile( const char *filename, missileAttributes_t *ma )
 #endif
 			defined |= IMPACT_FLESH_SND;
 		}
-		/*else
-		{
-			Log::Warn( "%s: unknown token '%s'\n", filename, token );
```

Comment:
I wonder why those warnings are disabled in the first place…

Reply:
so, shuold I let this dead code around?